pages/index.vue

```diff
@@ -16,6 +16,18 @@ const latestArticles = await findArticlePages({
 const articles = await findArticlePages('').then(({ data }) =>
   normalizeFindArticlePagesResponse(data)
 )
+const articlesToShow = ref(6)
+
+const homePageCollections = []
+const homePageCollectionItems = await findPage('/').then(({ data }) =>
```

Comment:
it says this is not being used anywhere.

Reply:
I guess we don't need that! I removed it :)